packages/gatsby-theme-docs/src/layouts/internals/layout-header.js

```diff
@@ -224,14 +273,25 @@ const LayoutHeader = forwardRef((props, ref) => {
             onClick={props.toggleTopMenu}
             onKeyPress={handleTopMenuButtonKeyPress}
           >
-            <SpacingsInline alignItems="center">
-              <span>{props.siteTitle}</span>
-              {props.isTopMenuOpen ? (
-                <AngleUpIcon size="medium" color="info" />
-              ) : (
-                <AngleDownIcon size="medium" />
+            <TitleContainer hasContext={!!siteContextTitle}>
```

Comment:
Can you give an example when the site context title is not defined? Is it only for backwards compatibility? Either way we can maybe leave a comment explaining the use case.

Reply:
I implemented this as it was a requirement of the ticket:
https://github.com/commercetools/commercetools-docs/issues/2374

With the current setup I believe we'll always have the context available.  I'll add a comment to explain that